.drone.yml

```diff
@@ -375,6 +388,11 @@ services:
     when:
       event: [ push, tag, pull_request ]
 
+  tidb:
+    image: pingcap/tidb:v2.1.3
+    when:
```

Comment:
Need create database.

Reply:
This should create the DB: https://github.com/go-gitea/gitea/blob/master/integrations/integration_test.go#L113